consoles/sshVirtsh.pm

```diff
@@ -384,7 +384,14 @@ sub _copy_image_vmware ($self, $name, $backingfile, $file_basename, $vmware_open
       "echo File $file_basename is being copied by other process, sleeping for 60 seconds; sleep 60;" .
       'done;' .
       'else ' .
+      "REQUIRED_SPACE=\$(du -b /vmfs/volumes/$vmware_nfs_datastore/$nfs_dir/$file_basename | awk '{print \$1}'); " .
+      "AVAILABLE_SPACE=\$(df -B1 $vmware_openqa_datastore | awk 'NR==2 {print \$4}'); " .
+      "if [ \"\$REQUIRED_SPACE\" -gt \"\$AVAILABLE_SPACE\" ]; then " .
```

Comment:
it should be better wirh `-qe` . I am also thinking if it has to have a margin. if the `$REQUIRED_SPACE` take all the space after the operation, the disk will not have space to "breath", right?

Reply:
I found this https://thetechylife.com/how-much-space-should-you-leave-on-an-ssd/
not sure if you want to think it over. it is suggested a ~20%. I will leave it to others for taking the final decision about the handling